porting/configure_test_fix/USBModed.rst

```diff
@@ -0,0 +1,90 @@
+- IDPRODUCT_RNDIS_ADB
+
+It accepts the IDs in hexadecimal without ``0x`` prefix. The IDs could be found in:
+
+- Your device's ``init.<device>.rc`` or ``init.<chipset>.rc``.
+- Your device's USB HAL source code, if it happens to ship one.
+- Your device's ``setupusb`` script, if available.
+
+Failing that, you might have to probe how your device shows up when booted on Android.
+
+.. note::
+    We're aware that some device use different vendor IDs for different modes. If your device are one of those, please upvote `this issue on GitLab <https://gitlab.com/ubports/development/core/packaging/usb-moded/-/issues/5>`_.
+
+Devices' pretty name
+^^^^^^^^^^^^^^^^^^^^
+
+This is the name that will show up when plugged into a computer. Most of the time, we can detect this from Android properties, but if the detection is not correct or you would like the device to display differently, it can be configured here.
+
+- MANUFACTURER
+- PRODUCT
+
+usb-moded's arguments
+^^^^^^^^^^^^^^^^^^^^^
+
+This config controls the launching flag of the usb-moded daemon itself. Most of the time, usb-moded should work without any flag. However, the rootfs ships a default with ``-r`` which enables the rescue mode upon boot (see below). The behavior of the rescue mode can be confusing for end users, so it's recommended that porters config this value to the blank value to disable the rescue mode.
+
+- USB_MODED_ARGS
+
+About usb-moded's rescue mode
+-----------------------------
+
+usb-moded's rescue mode is similar to hybris-usb's usb-tethering mode. It enables the phone's USB to function as a network interface, and starts a DHCP server. The phone will be available at 10.15.19.82, and an emergency SSH server will be available on port 8022. This is intended for porters to debug issues during the development, and thus it will stay in that mode until the cable is re-plugged.
+
+This behavior is helpful for the porters. However, this can be confusing to end-users when the port is shipped. So I recommended disabling this mode before making the port available to users by overriding ``USB_MODED_ARGS`` as discussed above.
+
+Another rescue feature available is the ability to force usb-moded to go to "rescue mode" or to enable ADB on every boot. This can be done by placing an empty file at ``/userdata/.force-ssh`` or ``/userdata/.force-adb`` (which translate to ``/data/.force-ssh`` or ``/data/.force-adb`` in the recovery), and the system will make sure the correct mode is enabled.
+
+Configuring usb-moded directly
+------------------------------
+
+Other than the configurators, porters can also set usb-moded options directly via ``/etc/usb-moded/90-device-specific-config.ini``. However, this is intended as an escape hatch in case the automatic detection fails, and if you found yourself need to use this file, consider filing issue over usb-moded's packaging repository. See `usb-moded's docs <https://github.com/sailfishos/usb-moded/blob/master/docs/usb_moded-doc.txt>` for the syntax.
```

Comment:
Maybe "Besides tweaking options for the Ubuntu Touch configurator script", took me some time to understand the difference before.

Reply:
Please take another look. I left the phrase "Ubuntu Touch" out to make it shorter. How does that look?

Reply:
Looks ok to me